docs/websockets.md

```diff
@@ -91,16 +91,6 @@ configuring individual backends for more information.
 
 ## Implementation-specific configuration
 
-### async-http-client based backends
```

Comment:
I think I would leave that - if somebody already uses the AHC backend, they might be looking for its config options

Reply:
Moved at the very bottom with a depreciated message.